### ISSUE TITLE
Not working with VIA

### ISSUE THREAD

**Issue by vndre** [konostore/Discord-Nitro#1]
There is an [open issue](https://github.com/the-via/releases/issues/269) in the VIA repo that affects all users of this keyboard: VIA doesn't connect to the keyboard so we can't make any modifications.

You can [create a PR](https://github.com/the-via/keyboards/pulls) to add Discord Nitro as a supported device for VIA, but I didn't find any related to this keyboard, I wonder why.

Anyways VIA's development doesn't seems very active (looking at the issues I would call it dead...) so I am feeling pretty pessimist about the possibilities left for us the owners :(

**Comment by Kamalaja** [comment]
For those who have the Nitro - here is what I have found in order to make it work. 

TLDR - Kono really did not do any QC on this, but its fixable.

**Issue**
In the VIA json files provided - the release version has the incorrect ProductID as well as the incorrect format - However the source code implementation is the correct JSON file, albiet with an incorrect ProductID

**Solution**
1. Download the source code zip file and get the JSON file from there
2. Update the ProductID to `0x0068`
3. Follow the Steps in the instructions. 
3a. I found no success on Linux - I had to go to VIA on windows to get it working. I believe it's a UDEV issue, and I am not competent enough to resolve.
3b. I found on Windows, I needed to make sure the keyboard was directly connected to the computer. I could not go through a hub of any kind. 

Hope this helps folks looking for a solution.
